### ISSUE TITLE
Hope to be able to show in the right place

### ISSUE THREAD

**Issue by eryajf** [markbattistella/docsify-sidebarFooter#3]
I use the following configuration

```
autoFooter: {
        name:       'Eryajf',          // company display name (required)
        copyYear:   2021,             // start copyright year (required)
        url:        '',          // company url (optional)
        policy:     true,   // show Privacy Policy (optional)
        terms:      true,   // show Terms of Service (optional)
        cookies:    false,   // show Cookies Policy (optional)
        onBody:     true             // if true it is on the main doc
      }
```

The following effects are obtained：

![image](https://user-images.githubusercontent.com/33259379/147908760-a960e157-ef71-4152-8ea9-f9edcaf85443.png)

How should I show it in the text。

**Comment by markbattistella** [comment]
Currently working on fixing it so that it can be easily adapted to sites particularly mobile.
Hopefully in a couple of weeks

**Comment by eryajf** [comment]
Thank you, continuous attention

**Comment by markbattistella** [comment]
@eryajf sorry for the delay - but this has been fixed in v5.0.1

There are some new options and some deprecated ones so please read the instructions carefully.
